syntax="proto3";

message PlayerLoginMsg{
	int32 msgType = 1;		// 消息类型
	string Username=2;		// 账户
	string UserPass=3;		// 密码
}

message RespondMsg
{
    bool rv = 1;			// 请求执行状态
    int32 msgType = 2;		// 消息类型
	bytes data = 5;			// 消息类容
}